### ISSUE TITLE
Automation actions

### ISSUE THREAD

**Issue by meenanader** [MilosKozak/AndroidAPS#2993]
Reporting bugs
--------------
- **Note the precise time the problem occurred** and describe the circumstances and steps that caused
  the problem
- Note the Build version (found in the About dialog in the app, when pressing the three dots in the
  upper-right corner).
- Obtain the app's log files, which can be found on the phone in
  _/storage/emulated/0/Android/data/info.nightscout.androidaps/_
  See https://github.com/MilosKozak/AndroidAPS/wiki/Accessing-logfiles
- Open an issue at https://github.com/MilosKozak/AndroidAPS/issues/new

Automation actions gets applied to the main screen but not applied to the pump for profile and percentage 
switching.
Using Medtronic pump driver.
It happens consistently, so it is not hard to replicate.
It is very dangerous when switching from high insulin profile to lower one as the actual action does not get applied and it gives you a fake feeling the action has been applied on the main screen.

**Comment by MilosKozak** [comment]
did you try apply exactly the same profile from menu?

**Comment by meenanader** [comment]
Yes, it works fine if I do it manually, but in automation it does not work, it gets the action to set profile but does not apply it.
That happens especially if you have an action which set profile with a percent, it seems to be quick and does not set the profile but only the percentage to the old set profile.
However on the main page it shows it has changed.
Currently I disabled all the automation and will do more investigation on what makes it to fail. If just profile or profile with percentage.
